src/styles/blocks/card.scss

```diff
@@ -0,0 +1,62 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  max-width: 200px;
+
+  padding: 32px 16px 16px;
+
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  border-radius: 5px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  color: $main-accent;
+  background-color: #fff;
+  &__image {
+    max-width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+  &__title {
+    margin-top: 40px;
```

Comment:
use padding to add gaps in your code, fix it everywhere please

Reply:
Why we can`t use margin? It`s outer space for elements.
In BEM we can use margin in Elements and Modificators

Reply:
It's better to use margins to add gaps between your containers, but not inside of your card

You can read more [here ](https://mailchimp.com/resources/padding-vs-margin/)

Reply:
It's okay to use `margin`, but it's better to use `margin` only for external spacing between blocks and to use `padding` for internal spacing.